pallets/marketplace/src/tests/migration.rs

```diff
@@ -87,13 +87,17 @@ mod v6 {
                 ALICE,
                 Vec::default(),
                 "Ternoa Marketplace".into(),
+                None,
```

Comment:
You can comment out the whole `v6 mod `because it's not going to be used anyway.